x-pack/plugins/apm/server/routes/latency_distribution/route.ts

```diff
@@ -78,10 +95,23 @@ const latencyOverallTransactionDistributionRoute = createApmServerRoute({
               (fieldValuePair): QueryDslQueryContainer[] =>
                 termQuery(fieldValuePair.fieldName, fieldValuePair.fieldValue)
             ) ?? []),
```

Comment:
This addresses feedback from the last PR https://github.com/elastic/kibana/pull/136083#discussion_r917924443. This filters metrics documents by those with existing duration field `'transaction.duration.histogram'` (from `getDurationField` util function).